### PR TITLE
add `scaffold_stylesheet` method to guide [ci skip]

### PR DESCRIPTION
**Pull request by y-yagi**
`scaffold_stylesheet` was added in #20479.

### CODE REVIEW

File: guides/source/configuring.md [hunk 1]
```diff
@@ -191,6 +191,7 @@ The full set of methods that can be used in this block are as follows:
 * `scaffold_controller` different from `resource_controller`, defines which generator to use for generating a _scaffolded_ controller when using `rails generate scaffold`. Defaults to `:scaffold_controller`.
 * `stylesheets` turns on the hook for stylesheets in generators. Used in Rails for when the `scaffold` generator is run, but this hook can be used in other generates as well. Defaults to `true`.
 * `stylesheet_engine` configures the stylesheet engine (for eg. sass) to be used when generating assets. Defaults to `:css`.
+* `scaffold_stylesheet` allows to create `scaffold.css` on generating a scaffold. Defaults to `true`.
```
Comment (maclover7) [inline guides/source/configuring.md]: `scaffold_stylesheet` creates `scaffold.css`

**Comment by maclover7** [comment]
One small grammar comment, then ready to merge. :)

**Comment by y-yagi** [comment]
Thanks for reviewing! I updated.

File: guides/source/configuring.md [hunk 2]
```diff
@@ -191,6 +191,7 @@ The full set of methods that can be used in this block are as follows:
 * `scaffold_controller` different from `resource_controller`, defines which generator to use for generating a _scaffolded_ controller when using `rails generate scaffold`. Defaults to `:scaffold_controller`.
 * `stylesheets` turns on the hook for stylesheets in generators. Used in Rails for when the `scaffold` generator is run, but this hook can be used in other generates as well. Defaults to `true`.
 * `stylesheet_engine` configures the stylesheet engine (for eg. sass) to be used when generating assets. Defaults to `:css`.
+* `scaffold_stylesheet` creates `scaffold.css` on generating a scaffold. Defaults to `true`.
```
Comment (maclover7) [inline guides/source/configuring.md]: on --> when

scaffold --> scaffolded resource

**Comment by maclover7** [comment]
Sorry a few more comments :grimacing:

**Comment by y-yagi** [comment]
Thanks! I've updated again :)

**Comment by maclover7** [comment]
:+1: Thank you for your contribution!

**Comment by maclover7** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by y-yagi** [comment]
:smile:

**Pull request merged**